<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvenementEditor</class>
 <widget class="QDialog" name="EvenementEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Liste des événements programmés</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidgetEvenements">
     <column>
      <property name="text">
       <string>Titre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Début</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fin</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonSupprimer">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonModifier">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonAnnuler">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>EvenementEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
